### PR TITLE
Javalab: Finish the file menu and moving the editor tab menu to an accessible dropdown button 

### PR DESCRIPTION
**Pull request by JillianK**
A couple things in this PR:
- Add a file dropdown to the tabs that brings the selected file to the first tab and selects it
- Make the unselected tabs in light mode look less like links
- Move the "context menu" to a button dropdown on the selected tab, making it accessible

A demo of all of these:

https://user-images.githubusercontent.com/17147070/115768595-5e0c0a00-a35f-11eb-9a3f-5bc4ec3a4dbe.mov

## Links

- jira ticket: [for the file menu](https://codedotorg.atlassian.net/browse/CSA-259?atlOrigin=eyJpIjoiNjdjNTdjZDgwNjQwNDA3M2E4ZWZhN2U4YjNjYmU0MWYiLCJwIjoiaiJ9)
- jira ticket: [for making the tab menu accessible](https://codedotorg.atlassian.net/browse/CSA-260?atlOrigin=eyJpIjoiYzljNjVhNTVjZGQ5NGY0MWI2Njc3MjM5NjNhODZkMGQiLCJwIjoiaiJ9)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/JavalabEditorTabMenu.jsx [hunk 1]
```diff
@@ -29,36 +28,32 @@ const styles = {
  * A menu with a set of clickable links that calls the cancel handler if you
  * click outside the menu or the cancel button.
  */
-export const JavalabContextMenu = class JavalabContextMenuComponent extends Component {
+export const JavalabTabMenu = class JavalabTabMenuComponent extends Component {
```
Comment (molly-moen) [inline apps/src/javalab/JavalabEditorTabMenu.jsx]: do we need to export this? Or do we only ever use the default export at the bottom?

Comment (JillianK) [reply apps/src/javalab/JavalabEditorTabMenu.jsx]: Oops, we don't need to export it, I'll remove that and the same thing in the file explorer

File: apps/src/javalab/JavalabEditor.jsx [hunk 2]
```diff
@@ -25,6 +27,19 @@ const style = {
   },
   darkBackground: {
     backgroundColor: color.dark_slate_gray
+  },
+  fileMenuToggleButton: {
+    margin: '0px 0px 0px 4px',
```
Comment (jmkulwik) [inline apps/src/javalab/JavalabEditor.jsx]: NIT: I think you can just do `0, 0, 0, 4px` here

**Review by molly-moen — APPROVED** [review]
LGTM!

File: apps/src/javalab/JavalabEditor.jsx [hunk 3]
```diff
@@ -93,28 +112,34 @@ class JavalabEditor extends React.Component {
...
-    this.setState({
-      showMenu: true,
-      contextTarget: key,
-      menuPosition: {
-        top: `${boundingRect.bottom}px`,
-        left: `${boundingRect.left}px`
-      }
-    });
+  toggleTabMenu(key, e) {
+    if (key === this.state.contextTarget) {
```
Comment (jmkulwik) [inline apps/src/javalab/JavalabEditor.jsx:127]: NIT: I'd recommend adding a couple comments here and the other places. i.e. I was pretty confused if the tabs were "tab navigation" or "editor tabs" and whether the keys were "ids" or "keyboard hotkeys." :)

File: apps/src/javalab/JavalabFileExplorer.jsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+import color from '@cdo/apps/util/color';
+import onClickOutside from 'react-onclickoutside';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+const styles = {
```
Comment (jmkulwik) [inline apps/src/javalab/JavalabFileExplorer.jsx:8]: [non-blocking comment] There's a lot of styles here. Consider making this a css file instead. Here's a PR that uses a css file: https://github.com/code-dot-org/code-dot-org/pull/34679/files

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**